STRATEGIC-INITIATIVES.md

```diff
@@ -0,0 +1,45 @@
+# Strategic Initiatives
+
+At any one time the Node.js project has a number of strategic initiatives
+underway.  With the less-tangible approach to the work that the Community 
+Committee does, having a way to track this work is important.
+
+For each strategic initiative, the Community Committee's goal is to have a
+single point of reference - a champion - that understands what's happening
+with the initiative to the fullest extent posssible, and can report on the
+initiative as needed.
+
+A review of the initiatives will be a standing item on the Community Committee agenda (even
+if the update is 'nothing new') as a way to ensure they are active
+and have the support needed.
+
+# Current Initiatives
```

Comment:
Do you think it'd be worthwhile having a small description of each initiative just so people can get a good idea of it at a glance? Or should we just keep it brief.

Reply:
@rachelnicole definitely think that would be a good addition to the doc 👍

Reply:
@bnb I'll make a new PR this eve.

Reply:
@rachelnicole awesome 🙌